internal/aghtls/aghtls.go

```diff
@@ -28,3 +33,17 @@ func SaferCipherSuites() (safe []uint16) {
 
 	return safe
 }
+
+// ParseCipherIDs returns a set of cipher suites with the cipher names provided
+func ParseCipherIDs(ciphers []string) (userCiphers []uint16) {
+	for _, s := range tls.CipherSuites() {
+		if slices.Contains(ciphers, s.Name) {
+			userCiphers = append(userCiphers, s.ID)
+			log.Debug("user specified cipher : %s, ID : %d", s.Name, s.ID)
```

Comment:
I'm not sure that logging belongs in here at all.  It's better to log custom ciphers, if they are present, before parsing them.

Reply:
I think that logging should not be present in what is essentially a general-purpose function.  For some context, the current plan is that most of package `aghtls` will be imported into our `golibs` module one day.